pennylane/data/base/dataset.py

```diff
@@ -264,25 +290,27 @@ def __delattr__(self, __name: str) -> None:
         except KeyError as exc:
             raise AttributeError(f"'{type(self)}' object has no attribute '{__name}'") from exc
 
+    def __repr_items__(self) -> dict[str, str]:
+        return {**self.params, "attributes": str(list(self.attrs))}
```

Comment:
I've never seen `__repr_items__` before. Is it a standard?

Reply:
It's not standard, no. The idea is that subclasses can override it to add elements to the repr (e.g `<Dataset {item}={repr}, ...>` without having to re-write the whole `__repr__` method. But I don't think it's actually necessary, I'll remove it